arch/arm/src/cxd56xx/cxd56_sfc.c

```diff
@@ -101,7 +102,8 @@ static ssize_t cxd56_bread(FAR struct mtd_dev_s *dev, off_t startblock,
 {
   int ret;
 
-  finfo("bread: %08lx (%u blocks)\n", startblock << PAGE_SHIFT, nblocks);
+  finfo("bread: %" PRIxOFF "(%u blocks)\n",
```

Comment:
```suggestion
  finfo("bread: %" PRIxOFF " (%u blocks)\n",
```
I believe there is a missing whitespace here.

Reply:
Done.